recipes/gnu-getopt/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "gnu-getopt" %}
+{% set version = "1.1.6" %}
+{% set md5 = "4eaff9e6d8b2504a2e0d29ff0e9fd956" %}
+
+package: 
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz  # [osx]
+  md5: {{ md5 }}  # [osx]
+  url: http://frodo.looijaard.name/system/files/software/getopt/getopt-{{ version }}.tar.gz  # [osx]
+  patches:
+    - Makefile.osxpatch  # [osx]
+
+build:
+  number: 0
+  skip: True  # [not osx]
+
+requirements:
+  build:
+    - gcc  # [osx]
+    - gettext  # [osx]
+  run:
+    - libgcc  # [osx]
+    - gettext  # [osx]
+
+test:
+    commands:
+        - '$PREFIX/bin/gnu-getopt -V'  # > /dev/null'
```

Comment:
Single quotes don't seem like a good idea here given the environment variable.

Reply:
The quote variant shouldn't matter since the string should hit the yaml parser first, no? Changing them out I can remove the comment though...